test/fail_compilation/diag_dip1000.d

```diff
@@ -0,0 +1,26 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/diag_dip1000.d(13): Error: Top-level function `testScope` has no `this` to which `scope` can apply
+fail_compilation/diag_dip1000.d(14): Error: Top-level function `testReturn` has no `this` to which `return` can apply
```

Comment:
Sorry to nitpick, but that's the only error message with uppercase at the beginning of the message.